app/models/extractors/question_extractor.rb

```diff
@@ -19,8 +22,29 @@ def extract_data_for(classification)
 
     private
 
+    def fetch_annotations(classification)
```

Comment:
not a blocker - this is a little hard to read / verbose imho, maybe this is better...
``` ruby
annotations = classification.annotations.fetch(task_key)
rescue KeyError
  if if_missing == "error"
    raise MissingAnnotation, "No annotations for task #{task_key}"
  elsif if_missing == "ignore"
    annotations = []
  end
end
annotations
```

Reply:
Nice!